Firestore/Source/API/FIRDocumentReference.mm

```diff
@@ -313,4 +298,28 @@ - (FSTListenOptions *)internalOptions:(nullable FIRDocumentListenOptions *)optio
 
 @end
 
+#pragma mark - FIRDocumentReference (Internal)
+
+@implementation FIRDocumentReference (Internal)
+
++ (instancetype)referenceWithPath:(const ResourcePath &)path firestore:(FIRFirestore *)firestore {
+  if (path.size() % 2 != 0) {
+    FSTThrowInvalidArgument(
+        @"Invalid document reference. Document references must have an even "
+         "number of segments, but %s has %zu",
```

Comment:
It's a C99 specifier for `size_t`

Reply:
`git grep "%zu"` returns other references; I'd believe it works.

Reply:
Thanks.

Apparently VS supports it since 2015, so no concern here. Perhaps the same specifier should then be used for printing size in `FSTSpecTests.mm`.